### ISSUE TITLE
Does Redisson 3.24.1 supports, AWS Elastic Redis 7.x version? OR which version of Redisson supports AWS Elastic Redis 7.x?

### ISSUE THREAD

**Issue by rishiCS** [redisson/redisson#5700]

**Comment by mrniko** [comment]
Yes, it does. Let me know if you have any issues.

**Comment by rishiCS** [comment]
Thanks @mrniko , So I won't need to update my Redisson client to latest?

**Comment by mrniko** [comment]
No

**Comment by rishiCS** [comment]
@mrniko Does Redisson supports [exponential-backoff-strategy](https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/BestPractices.Clients.Redis.Discovery.html) on retry?

**Comment by mrniko** [comment]
@rishiCS 

Yes, it does https://redisson.pro/docs/fault-tolerance-and-recovery/#reconnection-settings
